test/module/irohad/multi_sig_transactions/mst_test_helpers.hpp

```diff
@@ -46,8 +46,12 @@ auto addSignatures(Batch &&batch, int tx_number, Signatures... signatures) {
     batch->addSignature(tx_number, sig_pair.first, sig_pair.second);
   };
 
+  // pack expansion trick:
+  // generate array with zeros. Operator , express left lambda and return right
```

Comment:
-> an ellipsis applies insert_signatures to each signature, operator comma returns the rightmost argument, which is 0